tests/integration/pilot/tunneling/forward-proxy/destination-rule.tmpl.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: networking.istio.io/v1alpha3
```

Comment:
these types of files should likely be under testdata/ which go has some special handling for

Reply:
Done